topostats/plotting.py

```diff
@@ -272,9 +276,62 @@ def _set_label(self, var: str):
         self.label = self.var_to_label[var]
         LOGGER.debug(f"[plotting] self.label     : {self.label}")
 
-    def summarise_by_image(self):
-        """Summarise statistics by image."""
-        self.summary_data = self.df.groupby(["image", "threshold"]).describe()
+    @staticmethod
+    def get_paths_relative_to_deepest_common_path(paths: list) -> list:
```

Comment:
Can be more explicit here with the type hint of `List[Path]` which says its a list of `Path` objects.

Reply:
Thank you! I did not know about this 😄